src/TodoListItem.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import TodoList from "./TodoList";
+const TodoListItem = (props) => {
+  return (
+    <div>
+      <h1>{props.data}</h1>
+      <ul>
+        {props.items.map((item, i) => {
+          return <li key={i}>{item.title}</li>;
+        })}
+      </ul>
```

Comment:
This <ul> should still be in `TodoList`. This component should only be concerned with one todo.
<img width="404" alt="202211_0831AM-Code" src="https://user-images.githubusercontent.com/10133316/199720970-3a2d3bc4-d700-47f4-9c4d-78e6029647a9.png">